citesphere/src/main/java/edu/asu/diging/citesphere/web/user/AsyncUpdateCitationsProcessor.java

```diff
@@ -0,0 +1,88 @@
+package edu.asu.diging.citesphere.web.user;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.social.zotero.api.ZoteroUpdateItemsStatuses;
+import org.springframework.social.zotero.exception.ZoteroConnectionException;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import edu.asu.diging.citesphere.core.exceptions.CitationIsOutdatedException;
+import edu.asu.diging.citesphere.core.exceptions.ZoteroHttpStatusException;
+import edu.asu.diging.citesphere.core.service.impl.AsyncCitationManager;
+import edu.asu.diging.citesphere.model.bib.ICitation;
+import edu.asu.diging.citesphere.user.IUser;
+
+@Component
+public class AsyncUpdateCitationsProcessor {
```

Comment:
I would switch the naming. Call this class `AsyncCitationManager` and the manager `AsyncUpdateCitationsProcessor`. It would be confusing that classes use the `CitationManager` but not the `AsyncCitationManager` directly. Making this class `AsyncCitationManager`, makes it easy to understand that in one case methods are run asynchronically and in the other they are not.